### DISCUSSION TITLE
Support for react 18

### DISCUSSION THREAD

**Discussion by munsim** [benhowell/react-grid-gallery#234] [Q&A] [answered]
Hi. Library doesn't seem to be working in react 18. Is there any work around? Cheers

**Comment by itoldya** [comment, marked as answer]
Hi! Our examples use react@18 and it seems to be working https://benhowell.github.io/react-grid-gallery/examples/custom-overlay. 

What error are you getting? Could you provide a minimal, reproducible example?
